src/python/install.sh

```diff
@@ -454,6 +454,16 @@ if [[ "${INSTALL_PYTHON_TOOLS}" = "true" ]] && [[ $(python --version) != "" ]];
             echo "${util} already installed. Skipping."
         fi
     done
+    
+    # Temporary: Removes “setup tools” metadata directory due to https://github.com/advisories/GHSA-r9hx-vwmv-q579
+
+    VULNERABLE_VERSIONS=("3.10" "3.11")
+    RUN_TIME_PY_VER_DETECT=$(python --version 2>&1)
+    PY_MAJOR_MINOR_VER=${RUN_TIME_PY_VER_DETECT:7:4};
+    if [[ ${VULNERABLE_VERSIONS[@]} =~ $PY_MAJOR_MINOR_VER ]]; then
```

Comment:
https://github.com/devcontainers/features/actions/runs/7746776660/job/21125752547?pr=815#step:4:145

Can you fix the linter errors?

Reply:
linter errors have been fixed. Two other cases fail but those are known fail issues. 👍